### PR TITLE
added new to type stub + fixed type hinting

### PR DESCRIPTION
**Pull request by hh-space-invader**
### All Submissions:

* [x] Contributions should target the `dev` branch. Did you create your branch from `dev`?
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you installed `pre-commit` with `pip3 install pre-commit` and set up hooks with `pre-commit install`?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by hh-space-invader** [comment]
#696
#687

**Comment by joein** [comment]
other than the comments, looks fine

**Review by joein — CHANGES_REQUESTED** [review]

**Comment by joein** [comment]
could you please regenerate poetry.lock

File: qdrant_client/conversions/common_types.py [hunk 1]
```diff
@@ -62,16 +62,16 @@ def get_args_subscribed(tp):  # type: ignore
     rest.PayloadSchemaType, rest.PayloadSchemaParams, int, grpc.PayloadIndexParams
 ]  # type(grpc.PayloadSchemaType) == int
 PointStruct: TypeAlias = rest.PointStruct
-Points = Union[rest.Batch, List[Union[rest.PointStruct, grpc.PointStruct]]]
+Points = Union[rest.Batch, Sequence[Union[rest.PointStruct, grpc.PointStruct]]]
```
Comment (joein) [inline qdrant_client/conversions/common_types.py:65]: `LocalCollection.upsert` should become `Union[Sequence[models.PointStruct], models.Batch]` then

**Review by joein — CHANGES_REQUESTED** [review]

**Review by joein — APPROVED** [review]

**Pull request merged**